twiggy/tests/all/expectations/paths_error_test_no_max_paths_csv

```diff
@@ -0,0 +1,9 @@
+Name,ShallowSize,ShallowSizePercent,Path
+std::io::error::Error::new::h8c006d5367bc92ed,339,0.7494694021931375,func[15] -> std::io::error::Error::new::h8c006d5367bc92ed
+func[15],1,0.0022108241952599928,std::io::impls::<impl std::io::Write for &'a mut W>::write::h5d7e5ba58acd05fd -> std::panicking::LOCAL_STDERR::__getit::h7827294b3348067a -> func[15]
+std::io::impls::<impl std::io::Write for &'a mut W>::write::h5d7e5ba58acd05fd,311,0.6875663247258578,func[33] -> std::io::impls::<impl std::io::Write for &'a mut W>::write::h5d7e5ba58acd05fd
+func[33],1,0.0022108241952599928,elem[0] -> func[33]
+elem[0],59,0.13043862752033958,
+std::panicking::LOCAL_STDERR::__getit::h7827294b3348067a,16,0.035373187124159884,func[37] -> std::panicking::LOCAL_STDERR::__getit::h7827294b3348067a
+func[37],1,0.0022108241952599928,elem[0] -> func[37]
+elem[0],59,0.13043862752033958,
```

Comment:
This expectation file is a good example of duplicate rows in the CSV output. We can clean that up separately :)